src/styles/_variables.scss

```diff
@@ -0,0 +1,6 @@
+$cardWidth: 200px;
+$gapX: 50px;
+$gapY: 48px;
+$padingX:40px;
+$padingY:50px;
+$base-color: #00acdc;
```

Comment:
I think in this case you need use one style for naming variables. CamelCase like in other variables, or change naming using kebab-case.